Specs/MiddlewareLib/0.8.0/MiddlewareLib.podspec.json

```diff
@@ -8,8 +8,8 @@
   },
   "authors": {"Taqtile": "contato@taqtile.com"},
   "source": {
-    "git": "https://github.com/indigotech/br-npc-middleware-lib-java/",
-    "tag": "0.8.0"
+    "git": "git@github.com:indigotech/br-npc-middleware-lib-java.git",
```

Comment:
@vnavarro 
why did you change to SSH URL now?
It's better to use HTTPS instead.